exercises/sum.cpp

```diff
@@ -6,3 +6,33 @@
   Insert the second number: 2
   Sum: 3
 */
+#include <iostream>
+using namespace std;
+
+//functions prototype
+
+float sum(float a, float b);
+
+//Main
+int main(void)
+{
+  float first_in, second_in, result;
+
+  cout << "Insert the first value: ";
+  cin >> first_in;
+  cout << "Insert the second value: ";
+  cin >> second_in;
+  result = sum(first_in, second_in);
+  cout<< "the result is " << result;
+  cout<< "\n";
+
+  return 0;
```

Comment:
potresti sostituire 0 con EXIT_SUCCESS in modo da migliorare la leggibilità del codice
```c++
#define EXIT_SUCCESS 0
...
return EXIT_SUCCESS;
```